core/README.md

```diff
@@ -0,0 +1,37 @@
+# math-lib-core
+
+This repository provides a `Math` & `Utils` modules which export the following functions.
+
+`Math` module
+1. `isqrt`     - `nat -> nat`
+2. `power`     - `nat * nat -> nat`
```

Comment:
General remark: "Should we stay with uncurrified power?"

Reply:
Yeah I feel we should stay with uncurrified because calling functions from jsligo, pasligo & (maybe in the future pyligo) will be idiomatic, otherwise we'll have to do `power(a)(b)` in jsligo which feels little weird.

But the down side is uncurried functions is not idiomatic in cameligo, so it's kind of a tradeof 3 vs 1

What do you feel?